public/assets/css/componentStyle.css

```diff
@@ -154,27 +154,50 @@ button.copy-button {
 	height: 1em;
```

Comment:
We ought to consider adopting a CSS class naming standard. I'm a fan of BEM http://getbem.com

This is non-blocking, but should probably be done before a large HTML/CSS change (such as the re-design).

Reply:
agreed